eng/common/scripts/Verify-Links.ps1

```diff
@@ -134,88 +133,135 @@ function ParseLinks([string]$baseUri, [string]$htmlContent)
+  #       LogWarning "[$statusCode] broken link $linkUri"
+  #       $linkValid = $false
+  #     }
+  #     else {
+  #       if ($null -ne $statusCode) {
+  #         Write-Host "[$statusCode] while requesting $linkUri"
+  #       }
+  #       else {
+  #         Write-Host "Exception while requesting $linkUri"
+  #         Write-Host $_.Exception.ToString()
+  #       }
+  #     }
+  #   }
+  # }
   $linkValid = $true
-  Write-Verbose "Checking link $linkUri..."  
-
-  if ($linkUri.IsFile) {
-    if (!(Test-Path $linkUri.LocalPath)) {
-      LogWarning "Link to file does not exist $($linkUri.LocalPath)"
-      $linkValid = $false
-    }
-  }
-  else {
-    try {
-      $headRequestSucceeded = $true
-      try {
-        # Attempt HEAD request first
-        $response = Invoke-WebRequest -Uri $linkUri -Method HEAD
-      }
-      catch {
-        $headRequestSucceeded = $false
-      }
-      if (!$headRequestSucceeded) {
-        # Attempt a GET request if the HEAD request failed.
-        $response = Invoke-WebRequest -Uri $linkUri -Method GET
-      }
-      $statusCode = $response.StatusCode
-      if ($statusCode -ne 200) {
-        Write-Host "[$statusCode] while requesting $linkUri"
-      }
-    }
-    catch {
-      $statusCode = $_.Exception.Response.StatusCode.value__
-
-      if(!$statusCode) {
-        # Try to pull the error code from any inner SocketException we might hit
-        $statusCode = $_.Exception.InnerException.ErrorCode
-      }
-
-      if ($statusCode -in $errorStatusCodes) {
-        LogWarning "[$statusCode] broken link $linkUri"
-        $linkValid = $false
-      }
-      else {
-        if ($null -ne $statusCode) {
-          Write-Host "[$statusCode] while requesting $linkUri"
-        }
-        else {
-          Write-Host "Exception while requesting $linkUri"
-          Write-Host $_.Exception.ToString()
-        }
-      }
-    }
-  }
-  
   if ($checkLinkGuidance) {
+    $replacedOne = ""
     # Check if the url is relative links, suppress the archor link validation.
     if (!$linkUri.IsAbsoluteUri -and !$linkUri.ToString().StartsWith("#")) {
-      LogWarning "DO NOT use relative link $linkUri. Please use absolute link instead. Check here for more infomation: https://aka.ms/azsdk/guideline/links"
+      $replacedOne = ResolveUri $pageLink $linkUri.ToString() $true
+      $replacedOne = $replacedOne -replace "file:///C:/sdk/azure-sdk-for-python", "https://github.com/Azure/azure-sdk-for-python/blob/master"
+      LogWarning "DO NOT use relative link $linkUri. Suggest Link: $replacedOne."
       $linkValid = $false
     }
-     # Check if link uri includes locale info.
-    if ($linkUri -match $locale) {
-      LogWarning "DO NOT include locale $locale information in links: $linkUri. Check here for more information: https://aka.ms/azsdk/guideline/links"
-      $linkValid = $false
+    #  # Check if link uri includes locale info.
+    # if ($linkUri -match $locale) {
```

Comment:
Might also be worth leaving this check on and fixing any more of these you might find.

Reply:
The locale stuff fixed together on the start of the script to save the operations.
https://github.com/Azure/azure-sdk-for-python/pull/13847/files#diff-e9e5d0400c5b5662fb38c88a2d65b0bcR341

Reply:
We use the original one to test if any broken links left. By far, it seems enough